webservices/cxf/src/java/org/sakaiproject/webservices/SakaiScript.java

```diff
@@ -4392,4 +4392,84 @@ protected Map getServersByServerId(Map servers) {
         return serverByServerId;
     }
 
+    /**
+     * Check if a user exists (either as an account in Sakai or in any external provider)
+     *
+     * @param sessionid the id of a valid session
+     * @param userid    the internal user id
+     * @return true/false
+     */
+    @WebMethod
+    @Path("/checkForUserById")
+    @Produces("text/plain")
+    @GET
+    public boolean checkForUserById(
```

Comment:
I've added the admin restriction. Maybe this was not a very dangerous one, but really I think not admin user won't need this webservice.